README.md

```diff
@@ -1,2 +1,27 @@
 # sailor-nodejs
+
 The official elastic.io library for bootstrapping and executing for Node.js connectors.
+
+Minimal .env vars required:
+
+MONGO_URI=mongodb://localhost/elasticdb
+AMQP_URI=amqp://guest:guest@localhost:5672
+TASK_ID=task123
+STEP_ID=step_2
+
+Default values of .env vars:
+
+COMPONENT_PATH=../
+INCOMING_MESSAGES_QUEUE={TASK_ID}.{STEP_ID}.incoming
+OUTGOING_MESSAGES_QUEUE={TASK_ID}.{STEP_ID}.outgoing
+ERRORS_QUEUE={TASK_ID}.{STEP_ID}.errors
```

Comment:
I thought that there will be one queue for all errors.. just for simplicity. Why per step?